app/uk/gov/hmrc/agentsubscriptionfrontend/views/address_form_with_errors.scala.html

```diff
@@ -0,0 +1,73 @@
+@import uk.gov.hmrc.agentsubscriptionfrontend.config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import uk.gov.hmrc.play.views.html.helpers.input
+@import uk.gov.hmrc.agentsubscriptionfrontend.views.html.formErrorSummary
+@import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes.SubscriptionController
+@import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
+@import uk.gov.hmrc.agentsubscriptionfrontend.models.DesAddress
+@(detailsForm: Form[DesAddress], message: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
also rename name message to error messages to be a little clearer to what they are referring too ;)

Reply:
Also it turns out we don't need this but double check with charlie!